src/components/Jobs/RunningJobs/JobDetails/JobDetails.js

```diff
@@ -24,14 +24,17 @@ const formatTime = dateStr => {
   return <span>{datePart} {timePart} {todayPart}</span>;
 };
 
-// TODO: remove defaults for jobProfileInfo, hrId and runBy once backend is in place
 const JobDetails = props => {
   const { job } = props;
 
   const {
     jobProfileInfo,
     exportedFiles,
-    runBy,
+    hrId,
+    runBy: {
```

Comment:
Let's not mix runBy feature in this PR. Create another PR for it, or add all needed changes to make runBy field also visible.

Reply:
Yeah, good catch.